packages/mdc-side-sheet/README.md

```diff
@@ -1,4 +1,318 @@
+      <div><label><input type="checkbox" checked> Personal</label></div>
+      <div><label><input type="checkbox" checked> Projects</label></div>
+      <div><label><input type="checkbox"> Reminders</label></div>
+      <div><label><input type="checkbox"> Work</label></div>
+    </div>
+  </aside>
+
+  <div class="mdc-side-sheet-app-content">
+    App Content
+  </div>
+</body>
+```
+
+> Apply the `mdc-side-sheet-app-content` class to the sibling element after the side-sheet for the open/close animations to work.\
+
+#### Usage with Top App Bar
+
+In cases where the side-sheet occupies the full viewport height, some styles must be applied to get the dismissible side-sheet and the content below the top app bar to independently scroll and work in all browsers.
+
+In the following example, the `mdc-side-sheet__content` and `main-content` elements should scroll independently of each other. The `mdc-side-sheet--dismissible` and `mdc-side-sheet-app-content` elements should then sit side-by-side. The markup looks something like this:
+
+```html
+<body>
+  <aside class="mdc-side-sheet mdc-side-sheet--dismissible">
+    <div class="mdc-side-sheet__content">
+      <div class="mdc-side-sheet__content">
+        <div><label><input type="checkbox"> Events</label></div>
+        <div><label><input type="checkbox" checked> Personal</label></div>
+        <div><label><input type="checkbox" checked> Projects</label></div>
+        <div><label><input type="checkbox"> Reminders</label></div>
+        <div><label><input type="checkbox"> Work</label></div>
+      </div>
+    </div>
+  </aside>
+
+  <div class="mdc-side-sheet-app-content">
+    <header class="mdc-top-app-bar app-bar" id="app-bar">
+      <div class="mdc-top-app-bar__row">
+        <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
+          <a href="#" class="demo-menu material-icons mdc-top-app-bar__navigation-icon">menu</a>
+          <span class="mdc-top-app-bar__title">Dismissible Side Sheet</span>
+        </section>
+      </div>
+    </header>
+
+    <main class="main-content" id="main-content">
+      <div class="mdc-top-app-bar--fixed-adjust">
+        App Content
+      </div>
+    </main>
+  </div>
+</body>
+```
+
+The CSS to match it looks like this:
+
+```scss
+// Note: these styles do not account for any paddings/margins that you may need.
+
+body {
+  display: flex;
+  height: 100vh;
+}
+
+.mdc-side-sheet-app-content {
+  flex: auto;
+  overflow: auto;
+}
+
+.main-content {
+  overflow: auto;
+  height: 100%;
+}
+
+.app-bar {
+  position: absolute;
+}
+```
+
+The JavaScript to toggle the side-sheet when the navigation button is clicked looks like this:
+
+```js
+import {MDCTopAppBar} from "@material/top-app-bar";
+const topAppBar = MDCTopAppBar.attachTo(document.getElementById('app-bar'));
+topAppBar.setScrollTarget(document.getElementById('main-content'));
+topAppBar.listen('MDCTopAppBar:nav', () => {
```

Comment:
This would need to change to a click handler on the specific icon, since only the top-left icon has a dedicated Top App Bar event.